src/classes/HiFiCommunicator.ts

```diff
@@ -90,6 +91,18 @@ export class HiFiCommunicator {
     private _webRTCSessionParams?: WebRTCSessionParams;
     private _customSTUNandTURNConfig?: CustomSTUNandTURNConfig;
 
+    /**
+     * If the World coordinate system is NOT compatible with the HiFi coordindate frame used by the mixer              
+     * then configure a HiFiCoordinateFrameUtil to transform to and from HiFi-frame.
+     *
+     * The World-frame is compatible iff:  
+     * (1) It is right-handed
+     * (2) It uses the Y-axis (positive or negative, doesn't matter) for the UP direction.
+     *           
+     * For all other cases create a {@link WorldFrameConfiguration} and pass it to the HiFiCommunicator constructor.
+     */
+    private _coordFrameUtil?: HiFiCoordinateFrameUtil;
```

Comment:
It would be up to the Application dev to supply a `_coordFrameUtil` to the `HiFiCommunictor` or not.  If one is supplied then it will be passed to each `HiFiMixerSession` constructor.